osgeo_importer/inspectors.py

```diff
@@ -4,8 +4,8 @@
 
 from django import db
 from django.conf import settings
-import gdal
-import ogr
+from osgeo import gdal
```

Comment:
We don't actually need to fix those. The trick is to install the correct gdal version, so:

```
$ pip uninstall pygdal
$ pip install gdal==$(gdal-config --version)
```

My mistake, sorry :(